packages/astro/src/core/messages.ts

```diff
@@ -62,6 +62,29 @@ export function serverStart({
 }): string {
 	// PACKAGE_VERSION is injected at build-time
 	const version = process.env.PACKAGE_VERSION ?? '0.0.0';
+
+	const messages = [
+		'',
+		`${bgGreen(bold(` astro `))} ${green(`v${version}`)} ${dim(`ready in`)} ${Math.round(
+			startupTime
+		)} ${dim('ms')}`,
+		'',
+		serverUrls({ resolvedUrls, host, base }),
+		`${dim('┃')} Press ${bold('h + enter')} for shortcuts`,
```

Comment:
Unfortunately this line doesn't fit well in this `|` element of the terminal UI. This is where the URLs are listed and the `| Label [...] URL` was designed with an intentional layout. The bold also pulls your attention to it more than it should (besides maybe the first time you see it, the URL is always the thing people want to see/copy/click 99% of the time).

For context, I'm going to be more purist than normal about this terminal UI for the next few months just because of how much work went into cleaning up and standardizing how info is presented. So if this sounds like a nit comment gone wrong, that's where I'm coming from :) 

One suggestion that i'd be okay with: what if we merge this PR, but remove all the changes made to `serverStart()`. Then we still get keyboard shortcuts in time for 4.0, but as more of an easter egg. Then, that buys us time to figure out the right way to show the help text in a future 4.1, 4.2, etc.

Otherwise, I'd push to block this PR until post 4.0 when we have some time to revisit this serverStart message formatting.

Reply:
Understandable! I tried fiddling a way to put this text but I couldn't find a nicer way too. I can revert this part then, and possibly the `u + enter` shortcut that displays the URLs.

I also don't mind pushing this post 4.0 since it shouldn't break anything, but I'll leave that to your call 😄

Reply:
I agree that the formatting is a bit weird. I'm going to take a pass at this!